packages/taiko-client/pkg/rpc/utils.go

```diff
@@ -31,6 +32,7 @@ var (
 		syscall.SIGTERM,
 		syscall.SIGQUIT,
 	}
+	ErrSlotBMarshal = errors.New("abi: cannot marshal in to go type: length insufficient 160 require 192")
```

Comment:
```suggestion
	ErrSlotBMarshal = errors.New("abi: cannot marshal into go type: length insufficient 160 require 192")
```

Reply:
It is copied from the error msg, so no need to change.